src/elements/common/logger/withLogger.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * @flow
+ * @file Wraps a component with the Logger context
```

Comment:
It doesn't look like we're using context anymore. Should this be updated?

Reply:
👍